ast-nodes/if-statement.dart

```diff
@@ -115,7 +115,72 @@ class IfStatement implements Statement, ScopeCreator {
   }
 
   Pointer<LLVMOpaqueValue> generateCode(Module module) {
-    // TODO: implement
-    return null;
+
+    Pointer<LLVMOpaqueValue> conditionValue = condition.generateCode(module);
+    if (conditionValue==null) {
+      return null;
+    }
+
+    Pointer<LLVMOpaqueValue> ifCond = llvm.LLVMConstICmp(
```

Comment:
Also I think you should be using `LLVMBuildICmp`, not const. Don't know yet what the const instruction is, but it doesn't seem like something we want